sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/DoublePointData.java

```diff
@@ -15,10 +17,20 @@
  */
 @Immutable
 @AutoValue
-public abstract class DoublePointData implements PointData {
+public abstract class DoublePointData implements SampledPointData {
+
   public static DoublePointData create(
       long startEpochNanos, long epochNanos, Attributes attributes, double value) {
-    return new AutoValue_DoublePointData(startEpochNanos, epochNanos, attributes, value);
+    return create(startEpochNanos, epochNanos, attributes, value, Collections.emptyList());
+  }
+
+  public static DoublePointData create(
```

Comment:
javadoc needed here.

Reply:
Interesting, previous code did NOT have javadoc here, added to both.